checkov/github/dal.py

```diff
@@ -85,28 +82,34 @@ def get_organization_webhooks(self) -> dict[str, Any] | None:
         return data
 
     def get_repository_collaborators(self) -> dict[str, Any] | None:
-        data = self._request(endpoint="repos/{}/{}/collaborators".format(self.org, self.current_repository),
-                             allowed_status_codes=[200])
+        if self.org:
+            endpoint = "repos/{}/{}/collaborators".format(self.org, self.current_repository)
+        else:
+            endpoint = "repos/{}/{}/collaborators".format(self.repo_owner, self.current_repository)
+        data = self._request(endpoint=endpoint, allowed_status_codes=[200])
         if not data:
             return None
         return data
 
     def persist_repository_collaborators(self) -> None:
         repository_collaborators = self.get_repository_collaborators()
         if repository_collaborators:
-            BaseVCSDAL.persist(path=self.github_repository_collaborators_file_path, conf=repository_collaborators)
+            BaseVCSDAL.persist(path=self.github_conf_file_paths["repository_collaborators"], conf=repository_collaborators)
 
     def get_repository_webhooks(self) -> dict[str, Any] | None:
-        data = self._request(endpoint="repos/{}/{}/hooks".format(self.org, self.current_repository),
-                             allowed_status_codes=[200])
+        if self.org:
```

Comment:
Same here, it is not needed, org can exist (to retrieve org security info) but the repository webhooks should be queried with repo-related details, i.e. repo_owner.

Reply:
there could be hooks for org, for repo of org, and for repo not under an org.
hence to get the correct hooks of the repo - there must be a specification if it's under a user or an org.
These 2 URLs will retrieve different data:
https://api.github.com/repos/marynaKK/terragoat/hooks - The private repo under my account
https://api.github.com/repos/KiteFlyingInTheSky/terragoat/hooks - The repo under the org 'KiteFlyingInTheSky', that is different from the previous repo.

It is not mentioned in the documentation, but I guess the owner could be the org or the user.

Reply:
If the user's org is also repo owner and he's interested in repo hooks under that org, he can provide org name as repo_owner env var.
The current solution will fail if I do provide org env var, but my org is not listed as repo owner (i.e. I'm the sole repo owner).

Reply:
Got it! - fixed.